### PR TITLE
throw error out when the key of mount.Options and EncryptOptions is the same

### PR DESCRIPTION
**Pull request by hahchenchen**
### Ⅰ. Describe what this PR does
throw error out when the key of mount.Options and EncryptOptions is the same

### Ⅱ. Does this pull request fix one issue?
fixes #2574 

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: pkg/ddc/alluxio/ufs_internal.go [hunk 1]
```diff
@@ -315,6 +316,11 @@ func (e *AlluxioEngine) genUFSMountOptions(m datav1alpha1.Mount, SharedOptions m
 func (e *AlluxioEngine) genEncryptOptions(EncryptOptions []datav1alpha1.EncryptOption, mOptions map[string]string, name string) (map[string]string, error) {
 	for _, item := range EncryptOptions {
 
+		if _, ok := mOptions[item.Name]; ok {
+			err := fmt.Errorf("the name %s is duplicated with the option", item.Name)
```
Comment (cheyang) [inline pkg/ddc/alluxio/ufs_internal.go:320]: I suggest advising the end user directly "the option %s is set more than two times, please double check the dataset's option and encryptOptions."

**Comment by cheyang** [comment]
@hahchenchen Would you mind providing the documentation about how to use it? thanks!

File: pkg/ddc/alluxio/ufs_internal.go (hunk 1 above)
Comment (hahchenchen) [reply pkg/ddc/alluxio/ufs_internal.go:320]: ok

Comment (hahchenchen) [reply pkg/ddc/alluxio/ufs_internal.go:320]: I think "more than one times", not "two times", right?

**Comment by hahchenchen** [comment]
> @hahchenchen Would you mind providing the documentation about how to use it? thanks!
Ok, the docs should be placed in docs/zh/samples and docs/en/samples directory, right?

File: pkg/ddc/alluxio/ufs_internal.go (hunk 1 above)
Comment (cheyang) [reply pkg/ddc/alluxio/ufs_internal.go:320]: I agree with "more than one times".

**Comment by cheyang** [comment]
> > @hahchenchen Would you mind providing the documentation about how to use it? thanks!
> > Ok, the docs should be placed in docs/zh/samples and docs/en/samples directory, right?

Yes, thanks.

**Review by cheyang — APPROVED** [review]

**Comment by cheyang** [comment]
@hahchenchen thanks for your great contribution!

**Pull request merged**